pkg/helm/charts/tests/centralservices/testdata/helmtest/declarative-configuration.test.yaml

```diff
@@ -0,0 +1,49 @@
+tests:
+- name: "Empty declarative configuration mounts shouldn't lead to additional volume mounts"
+  expect: |
+    [.deployments.central.spec.template.spec.containers[0].volumeMounts[]
+      | select(.mountPath == "/run/secrets/stackrox.io/declarative-configuration/")] | assertThat(length == 0)
+- name: "Multiple values set for declarative configuration mounts should lead to additional volume mounts"
+  values:
+    central:
+      declarativeConfiguration:
+        mounts:
+          - "some-declarative-config-1"
+          - "some-declarative-config-2"
+  expect: |
+    [.deployments.central.spec.template.spec.containers[0].volumeMounts[]
+      | select(.mountPath == "/run/secrets/stackrox.io/declarative-configuration/")] | assertThat(length == 2)
+    .deployments.central.spec.template.spec.containers[0].volumeMounts[]
+          | select(.name == "some-declarative-config-1") | assertThat(. != null)
+    .deployments.central.spec.template.spec.containers[0].volumeMounts[]
+          | select(.name == "some-declarative-config-1") | assertThat(.readOnly == true)
+    .deployments.central.spec.template.spec.containers[0].volumeMounts[]
+          | select(.name == "some-declarative-config-2") | assertThat(. != null)
+    .deployments.central.spec.template.spec.containers[0].volumeMounts[]
+          | select(.name == "some-declarative-config-2") | assertThat(.readOnly == true)
+    .deployments.central.spec.template.spec.volumes[] | select(.name == "some-declarative-config-1")
+     | assertThat(. != null)
+    .deployments.central.spec.template.spec.volumes[] | select(.name == "some-declarative-config-1")
+     | assertThat(.configMap.optional == true)
+    .deployments.central.spec.template.spec.volumes[] | select(.name == "some-declarative-config-2")
+     | assertThat(. != null)
+    .deployments.central.spec.template.spec.volumes[] | select(.name == "some-declarative-config-2")
+     | assertThat(.configMap.optional == true)
+
+- name: "Single value set for declarative configuration mound should lead to single additional volume mount"
```

Comment:
How does this test differ from multiple values test?

Reply:
I suppose not that much besides ensuring we can handle single and multiple values. If you think it's too much, I can remove, however I don't think there's much overhead of maintaining this test.